db/migrate/20200414123737_extract_tasks_source_ref_from_context.rb

```diff
@@ -1,4 +1,6 @@
 class ExtractTasksSourceRefFromContext < ActiveRecord::Migration[5.2]
+  class Task < ApplicationRecord; end
```

Comment:
I guess it is needed only when migrations and models are in different repos

Reply:
It is needed also in this repo as we can add various callbacks/validation or other surprises, which could affect data migrations unintentionally. This is reason why it is better to have defined models here in minimal version as much as possible.